open-vulnerability-clients/src/main/java/io/github/jeremylong/openvulnerability/client/epss/EpssItem.java

```diff
@@ -16,6 +16,7 @@
  */
 package io.github.jeremylong.openvulnerability.client.epss;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```

Comment:
```suggestion
```

Reply:
No such JSON schema exists for EPSS to my knowledge, so retained annotation as a "sensible default" for API consumption on the internet.

Reply:
no such JSON schema exists for EPSS to my knowledge so left the lenient parsing in place